shared/folders/files/index.js

```diff
@@ -83,10 +86,19 @@ class Files extends Component<void, Props, State> {
         title: 'Files',
         element: <ConnectedFiles path={currentPath.get('path')} />,
       },
+      subRoutes: {paperkey},
     }
   }
 }
 
+const injectRekey = (folder, tlfs) => {
```

Comment:
here we mix the solutions (devices which can fix the rekey problem) into the folder data